File: app/src/main/java/com/mcafi/calcetto/LogoutActivity.java
package com.mcafi.calcetto;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.google.firebase.auth.FirebaseAuth;

public class LogoutActivity extends AppCompatActivity implements View.OnClickListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_logout);
        Button Logout = findViewById(R.id.logout);
        Logout.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        FirebaseAuth.getInstance().signOut();
        startActivity(new Intent(LogoutActivity.this, LoginActivity.class));
    }
}

File: app/src/main/java/com/mcafi/calcetto/LogoutActivity.kt
package com.mcafi.calcetto

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.google.firebase.auth.FirebaseAuth

class LogoutActivity : AppCompatActivity(), View.OnClickListener {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_logout)
        val logout = findViewById<Button>(R.id.logoutButton)
        logout.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        FirebaseAuth.getInstance().signOut()
        startActivity(Intent(this@LogoutActivity, LoginActivity::class.java))
    }
}